app/src/main/java/com/atomicrobot/carbon/app/MainApplication.kt

```diff
@@ -20,12 +19,14 @@ open class MainApplication : Application() {
 
                 AppLogger()
 
+                val mods = Modules()
                 modules(
                     listOf(
-                        crashReporterModule,
-                        AppModule().appModule,
+                        mods.appModules,
+                        mods.dataModules,
+                        mods.viewModelModules,
```

Comment:
Probably spell out modules instead of mods (mods is typicallys short for "modifications".

Reply:
I originally had it that way, but opted for this because "modules" is a koin dsl key word / function (line 23). If you still prefer "modules" then I will change to that.

Reply:
Renamed to `mainModules` per Slack discussion.